migrations/orders/schema/20200211000003_create_electronic_orders_revisions.up.sql

```diff
@@ -1,22 +1,22 @@
 CREATE TABLE electronic_orders_revisions (
 	id uuid PRIMARY KEY,
-	electronic_order_id uuid,
+	electronic_order_id uuid NOT NULL,
 	FOREIGN KEY (electronic_order_id) REFERENCES electronic_orders (id) ON DELETE CASCADE,
-	seq_num INT DEFAULT 0,
-	given_name text,
+	seq_num INT DEFAULT 0 NOT NULL,
+	given_name text NOT NULL,
 	middle_name text NULL,
```

Comment:
`NULL` is the default, so we don't necessarily have to add it here.  Looking back at some of our `CREATE TABLE` migrations, it doesn't seem that we usually do.  But I'm fine with being explicit too as long as we're consistent within the same table like we are.

Reply:
I think that this is cruft from not knowing that `NULL` is the default but you have to set `NOT NULL`. It doesn't hurt anything but probably I should be explicit.